homework1/task2.py

```diff
@@ -0,0 +1,16 @@
+"""
+Given a cell with "it's a fib sequence" from slideshow,
+    please write function "check_fib", which accepts a Sequence of integers,
+    and returns if the given sequence is a Fibonacci sequence
+We guarantee, that the given sequence contain >= 0 integers inside.
+"""
+from typing import Sequence
+
+
+def check_fibonacci(data: Sequence[int]) -> bool:
+    if len(data) < 3:
```

Comment:
Well, it is a shortcut.
We guarantee, that the given sequence contain >= 0 integers inside.
So it might be 1 or 2 digits and they can be fibonacci slice.